src/routes/Books.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import Form from '../components/Form';
 import Book from '../components/Book';
+import '../styles/books.css';
 
 const Books = () => (
```

Comment:
[OPTIONAL] Unfortunately, same issue occurs here. You have no added fonts so system chose the default font family. I highly recommend you to add the related fonts to match Zeplin design.
<img width="460" alt="Screenshot 2023-03-04 at 8 13 52 PM" src="https://user-images.githubusercontent.com/25887253/222919833-ceec68bc-c7d2-43ed-a9ee-e14fe209a555.png">
<img width="1025" alt="Screenshot 2023-03-04 at 8 14 27 PM" src="https://user-images.githubusercontent.com/25887253/222919839-6651fa19-c0ef-4e2b-9e6f-ce356bef670b.png">

Reply:
Thank you. I will open an issue and address these optional changes after the merge. I appreciate the approval 🙏